Fuzzing/rstp.txt

```diff
@@ -0,0 +1,354 @@
+/
```

Comment:
Please make sure to remove all leading slashes, so as to not bloat the file unnecessarily.
For example, `/GetData.cgi` would become `GetData.cgi`